isPlainObject.js

```diff
@@ -31,14 +31,8 @@ function isPlainObject(value) {
   if (!isObjectLike(value) || getTag(value) != '[object Object]') {
     return false
   }
-  if (Object.getPrototypeOf(value) === null) {
-    return true
-  }
-  let proto = value
-  while (Object.getPrototypeOf(proto) !== null) {
-    proto = Object.getPrototypeOf(proto)
-  }
-  return Object.getPrototypeOf(value) === proto
+  const proto = Object.getPrototypeOf(proto)
```

Comment:
That will break _(Cannot access 'proto' before initialization)_. Did you mean `const proto = Object.getPrototypeOf(value)`?